realm/realm-annotations-processor/src/test/resources/io/realm/AllTypesRealmProxy.java

```diff
@@ -93,15 +95,13 @@
         this.columnInfo = (AllTypesColumnInfo) columnInfo;
     }
 
-    @Override
```

Comment:
Uh... the class file might not have the getters/setters before byte code manipulating....